server/src/metadata/workspace-migration/migrations/1697618021-addOpportunityTable.ts

```diff
@@ -14,7 +14,7 @@ export const addOpportunityTable: WorkspaceMigrationTableAction[] = [
     columns: [
       {
         columnName: 'amountAmountMicros',
-        columnType: 'integer',
+        columnType: 'bigint',
```

Comment:
don't we also want to update the columnType for all currency fields?

Reply:
I was wondering indeed, @Weiko think you are more aware to answer

Reply:
@martmull yes you'll need to update this file https://github.com/twentyhq/twenty/blob/main/server/src/metadata/field-metadata/utils/convert-field-metadata-to-column-action.util.ts#L128 as well as other existing migrations such as https://github.com/twentyhq/twenty/blob/main/server/src/metadata/workspace-migration/migrations/1697618009-addCompanyTable.ts#L58